haystack/pipelines/base.py

```diff
@@ -173,6 +174,9 @@ def load_from_yaml(cls, path: Path, pipeline_name: Optional[str] = None, overwri
             |        inputs: [MyESRetriever]
             ```
 
+        Note that, in case of a mismatch in version between Haystack and the YAML, a warning will be thrown.
```

Comment:
The verb thrown implies something like an exception. I'd rather go with shown/displayed/will appear, etc. What do you think?